doc/usage.rst

```diff
@@ -50,6 +48,15 @@ show [OPTION…] *PACKAGE*
 
    -s, --size              Print the size of each file
 
+validate [OPTION…] *PACKAGE* *SIZE*
+   Validate a partup PACKAGE
+
+   -s, --skip-checksums        Skip checksum verification for all input files
+   --size=SIZE                 Size of loop device. Default is 1G
+   -w, --write-image           Install the package to a file as an additional validation step
+   -b, --build-image=FILENAME  Build a bootable image from the package. Implies --write-image
+   --device=DEVICE             Validate against provided device
+
```

Comment:
Maybe add a short sentence informing about the partup version this is available in? Something like:
```
Available since: 1.1.0
```

Reply:
Very nitpicky, but could you move the "available since" string to the bottom of the `validate` paragraph (without parenthesis)? Like the GLib and other GNOME documentation does. :wink: